apps/challenges/permissions.py

```diff
@@ -21,3 +24,31 @@ def has_permission(self, request, view):
                 return False
         else:
             return False
+
+
+def is_challenge_creator(func):
```

Comment:
Ah I misunderstood the PR. I see I have to use the django-rest-framework permissions decorator. Sorry about that.